crates/cli/Cargo.toml

```
@@ -30,6 +30,7 @@ rouille = { version = "3.0.0", default-features = false }
 serde = { version = "1.0", features = ['derive'] }
 serde_derive = "1.0"
 serde_json = "1.0"
+shlex = "1.3.0"
```

```suggestion
shlex = "1"
```
Unless we *require at least* v1.3?